js/main.js

```diff
@@ -41,20 +53,25 @@ function setupPlayer() {
 		
 
 	var container = document.getElementById('player');
-	var player = new bitmovin.player.Player(container, conf);
+	// expose on window object for the key controller event handlers
+	window.player = new bitmovin.player.core.Player(container, conf);
 	
 	player.load(source).then(function(value) {
 		// Success
 		console.log("Successfully created bitmovin player instance");
+		player.buffer.setTargetLevel(bitmovin.player.core.BufferType.BackwardDuration, 10, bitmovin.player.core.MediaType.Video);
```

Comment:
You can already set the buffer levels before `load`. But why don't you do it directly in the config?

Reply:
I actually forgot about the buffer config and thought this was the only way to set the buffer levels now, will update